### DISCUSSION TITLE
Hard to find GitHub users to follow

### DISCUSSION THREAD

**Discussion by ghost** [community/community#30473] [Other Feature Feedback, Questions, & Ideas]
# ***You can't follow what you can't find.***

## ***Hard to do social media when the social part is missing.

With scrolling and clicking around, GitHub social media can be a huge time suck. But the following quick aspect is missing. We must find a way to engage with people on GitHub, starting with following people. If you know, you know how much a pain it is to find people you already know and follow them. ***

**Comment by queenofcorgis** [comment]
Hi there - thanks for the feedback! I moved this over to our Feed channel so it will have better visibility among the relevant teams 😄.
